### PR TITLE
Remove unused imports and variables

### PR DESCRIPTION
**Pull request by larsk21**

### CODE REVIEW

File: src/test/java/edu/kit/compiler/transform/IRExpressionVisitorTest.java [hunk 1]
```diff
@@ -46,7 +42,7 @@ private ExpressionNode getReturnExpressionFromProgram(String input) {
         for (ClassNode classNode: ast.getClasses()) {
             for (MethodNode.DynamicMethodNode m: classNode.getDynamicMethods()) {
                 var mapping = LocalVariableCounter.apply(m);
-                this.context = new TransformContext(
+                new TransformContext(
```
Comment (larsk21) [inline src/test/java/edu/kit/compiler/transform/IRExpressionVisitorTest.java:45]: Is this call necessary?

File: src/test/java/edu/kit/compiler/transform/TransformContextTest.java [hunk 2]
```diff
@@ -50,7 +45,7 @@ private MethodNode initMethod(String input) {
         this.stringTable = lexer.getStringTable();
         this.typeMapper = new TypeMapper(namespaceMapper, lexer.getStringTable());
         for (ClassNode classNode: node.getClasses()) {
-            ClassEntry entry = typeMapper.getClassEntry(classNode);
+            typeMapper.getClassEntry(classNode);
```
Comment (larsk21) [inline src/test/java/edu/kit/compiler/transform/TransformContextTest.java:48]: Is this call necessary?

File: src/test/java/edu/kit/compiler/transform/IRExpressionVisitorTest.java (hunk 1 above)
Comment (larsk21) [reply src/test/java/edu/kit/compiler/transform/IRExpressionVisitorTest.java:45]: @0xNB

File: src/test/java/edu/kit/compiler/transform/TransformContextTest.java (hunk 2 above)
Comment (larsk21) [reply src/test/java/edu/kit/compiler/transform/TransformContextTest.java:48]: @N-Maas

Comment (N-Maas) [reply src/test/java/edu/kit/compiler/transform/TransformContextTest.java:48]: No, I think this is a pure function anyway

File: src/test/java/edu/kit/compiler/transform/IRExpressionVisitorTest.java (hunk 1 above)
Comment (N-Maas) [reply src/test/java/edu/kit/compiler/transform/IRExpressionVisitorTest.java:45]: I don't think so. Actually, the whole file seems quite unnecessary to me.

**Review by N-Maas — APPROVED** [review]

**Pull request merged**